nservicebus/messaging/delayed-delivery.md

```diff
@@ -13,6 +13,12 @@ The message doesn't have to be dispatched immediately after sending, it can be d
 
 NOTE: Only send operations can be deferred. Publish and reply operations cannot be deferred.
 
+## Handling current message later
+To handle the current message at a later time `bus.HandleCurrentMessageLater()` method can be used.  This was the initial way of *deferring* a message. This would put the message at the end of the queue and the message eventually will be pick up again once other messages are processed. The problem with this approach is that if there are no other messages in the queue (or that many of them) the message will pop right back up so the intended deferring might not happen. 
+
+Note: RabbitMQ tries to put the message back in the queue as close to its original position as possible, which may mean at the head of the queue where it will be picked up immediately again. 
```

Comment:
If this is handled by by issuing a new send, then yes, I would not expect there to be any special behavior for RabbitMQ. It would be a new message arriving at the queue, and it would go to the end of the line.

If this was instead being handled in a way that was telling the transport to reject the message, then yes, it would be at the head of the queue like the note is indicating.